src/client/components/pages/entity-revisions.js

```diff
@@ -83,6 +105,11 @@ class EntityRevisions extends React.Component {
 		);
 	}
 
+/**
+* Renders the 'Revision History' column of the EntityRevisions page
```

Comment:
I'd have something like this here: "Renders the EntityRevisions page, which is a list of all the revisions made to an entity, along with information about the author and the first revision note for each revision"